### PR TITLE
Feature/initial xamarin app

### PR DESCRIPTION
**Pull request by tmaciulis22**
Migrated Login Screen, but there are still some things unfinished due to other migrations. After other migrations this screen should be fully completed.

### CODE REVIEW

File: app-android-xamarin/Moodis/Moodis/Extensions/IListExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moodis.Extensions
+{
+    public static class IListExtensions
+    {
+        public static bool IsNullOrEmpty<T>(this IList<T> list)
+        {
+            if(list == null || list.Count == 0)
```
Comment (andriusbusma) [inline app-android-xamarin/Moodis/Moodis/Extensions/IListExtensions.cs]: I think you could write return list == null || list.Count == 0 instead this whole block

Comment (tmaciulis22) [reply app-android-xamarin/Moodis/Moodis/Extensions/IListExtensions.cs]: Finally you started to review PRs !!! 🔥💯🔥

File: app-android-xamarin/Moodis/Moodis/Feature/SignIn/SignInViewModel.cs [hunk 2]
```diff
@@ -1,80 +1,81 @@
...
 
-        //public bool Authenticate(string username, string password)
-        //{
-        //    fetchUserList();
+        public bool Authenticate(string username, string password)
+        {
+            FetchUserList();
 
-        //    currentUser = userList.Find(user => user.username == username && user.password == Crypto.CalculateMD5Hash(password));
+            //currentUser = userList.Find(user => user.username == username && user.password == Crypto.CalculateMD5Hash(password));
```
Comment (andriusbusma) [inline app-android-xamarin/Moodis/Moodis/Feature/SignIn/SignInViewModel.cs]: Delete unused line

File: app-android-xamarin/Moodis/Moodis/Extensions/IListExtensions.cs (hunk 1 above)
Comment (andriusbusma) [inline app-android-xamarin/Moodis/Moodis/Extensions/IListExtensions.cs]: I think you could write return list == null || list.Count == 0 instead this whole block

**Review by andriusbusma — CHANGES_REQUESTED** [review]

**Review by Pilkas-git — APPROVED** [review]

**Review by AArlauskas — APPROVED** [review]
Šaunuolis, brolau!

**Pull request merged**